package io.swagger.api.exception;

public class UserNotFoundException extends RuntimeException {
}